amivapi/media.py

```diff
@@ -10,6 +10,7 @@
 
 """
 
+from bson import tz_util
```

Comment:
I basically used the tz from bson, because eve also did. Interesting question. I guess eve uses it, because they don't want to depend on pytz. However that is bad anyway, as eve depends on bson in that way, even if the mongo data layer is not used.
I am not sure whether there is any difference apart from what dependencies to libraries are generated.